mqtt/build/linux/cross.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
Do you think it is possible to combine build.sh and cross.sh into one? I noticed we have `CARGO` variable in the script 
https://github.com/Azure/iotedge/blob/master/mqtt/build/linux/build.sh#L23
so I assume we can use it like this
```bash
CARGO=cross ./build.sh
```

Reply:
Good point. I combined build.sh and test.sh